taxi_service/urls.py

```diff
@@ -22,4 +23,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
+    # path("__debug__/", include(debug_toolbar.urls)),
```

Comment:
The path for `debug_toolbar` is commented out. If you are using the Django Debug Toolbar for development, you should uncomment this line to enable its URL routing.